Documentation Changes for Teams and Workspaces 1.0 (#959)
100mslive/100ms-docs: docs/server-side/v2/introduction/webhook.mdx

## How to configure Webhook

> Webhook can be configured using [Developer](https://dashboard.100ms.live/developer) section of [100ms Dashboard](https://dashboard.100ms.live/). Webhook config exists at the workspace level and hence each workspace can be configured for its respective webhook URL. 

Reviewer: ```suggestion
Webhook can be configured using [Developer](https://dashboard.100ms.live/developer) section of [100ms Dashboard](https://dashboard.100ms.live/). Each workspace can have its own webhook configuration. 
```
